subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/ui/SubscriptionSettingsViewModel.kt

```diff
@@ -83,7 +83,7 @@ class SubscriptionSettingsViewModel @Inject constructor(
         val account = subscriptionsManager.getAccount() ?: return
         val subscription = subscriptionsManager.getSubscription() ?: return
 
-        val formatter = SimpleDateFormat("MMMM dd, yyyy", Locale.getDefault())
+        val formatter = DateFormat.getInstanceForSkeleton("ddMMMMyyyy")
```

Comment:
`17 December 2024` could feel out of place for a user based in US, couldn't it? What if picked the date format based on whether the current locale is `Locale.US`?

Reply:
Never mind, I looked into the documentation of the function :)

Reply:
We won't show `17 December 2024` to an US-based user. This approach chooses the most appropriate date format for the device locale, so for `en_US` we will still use `MMMM dd, yyyy`.